services/fxaccounts/FxAccountsKeys.jsm

```diff
@@ -191,10 +223,35 @@ class FxAccountsKeys {
     });
   }
 
+  /**
+   * @param {String} scopes Space separated requested scopes
+   * @param {String} clientId oauth client id
+   */
+  async getScopedKeys(scopes, clientId) {
+    const scopedKeys = {};
+    if (this._fxia.rustFxa) {
+      for (const scope of scopes.split(" ")) {
+        const { key } = await this._fxia.rustFxa.getAccessToken(scope);
```

Comment:
Hrm, if we don't already have an OAuth token with that scope, this is going to create one for no reason. I guess that's proably OK in practice because we'll usually have one already, but still, it's a shame.